<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdfSettings</class>
 <widget class="QDialog" name="PdfSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Select the options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="baseCheckBox">
          <property name="text">
           <string>Base</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ratingsCheckBox">
          <property name="text">
           <string>Ratings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="storagLocCheckBox">
          <property name="text">
           <string>Storage location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="purchaseCheckBox">
          <property name="text">
           <string>Purchase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="seriesCheckBox">
          <property name="text">
           <string>Series</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="transCheckBox">
          <property name="text">
           <string>Translation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="loanedCheckBox">
          <property name="text">
           <string>Loaned</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lentCheckBox">
          <property name="text">
           <string>Lent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="saleCheckBox">
          <property name="text">
           <string>Sale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sourceCheckBox">
          <property name="text">
           <string>Source of origin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="additivesCheckBox">
          <property name="text">
           <string>Additives</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="descrCheckBox">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
